Build failure: openloco

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

1. Build openloco
2. Get error

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/295019126

### Relevant log output

```console
Running phase: unpackPhase
unpacking source archive /nix/store/v8p2piz8zlrxwrk4gzlpmqx0rl8zaavp-source
source root is source
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/40p7d41c40ifa39iybdaddrlbaws7vxc-openloco-25.02/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/40p7d41c40ifa39iybdaddrlbaws7vxc-openloco-25.02/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/40p7d41c40ifa39iybdaddrlbaws7vxc-openloco-25.02/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/40p7d41c40ifa39iybdaddrlbaws7vxc-openloco-25.02/share/doc/openloco -DCMAKE_INSTALL_INFODIR=/nix/store/40p7d41c40ifa39iybdaddrlbaws7vxc-openloco-25.02/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/40p7d41c40ifa39iybdaddrlbaws7vxc-openloco-25.02/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/40p7d41c40ifa39iybdaddrlbaws7vxc-openloco-25.02/include -DCMAKE_INSTALL_SBINDIR=/nix/store/40p7d41c40ifa39iybdaddrlbaws7vxc-openloco-25.02/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/40p7d41c40ifa39iybdaddrlbaws7vxc-openloco-25.02/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/40p7d41c40ifa39iybdaddrlbaws7vxc-openloco-25.02/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/awp4diicbigk3xgsrs26jwx59g09hl43-gcc-wrapper-14.2.1.20250322/bin/strip -DCMAKE_RANLIB=/nix/store/awp4diicbigk3xgsrs26jwx59g09hl43-gcc-wrapper-14.2.1.20250322/bin/ranlib -DCMAKE_AR=/nix/store/awp4diicbigk3xgsrs26jwx59g09hl43-gcc-wrapper-14.2.1.20250322/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/40p7d41c40ifa39iybdaddrlbaws7vxc-openloco-25.02 -DOPENLOCO_BUILD_TESTS=NO
-- Could NOT find CCache (missing: CCache_EXECUTABLE) 
Hint: The project() command has not yet been called.  It sets up system-specific search paths.
-- The CXX compiler identification is GNU 14.2.1
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/awp4diicbigk3xgsrs26jwx59g09hl43-gcc-wrapper-14.2.1.20250322/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /nix/store/8zplvfpfd4kiq84m4ah7j765xxp4lb13-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found ZLIB: /nix/store/fkksm6bcsd7502ghx5lzmrj9cl7cqck1-zlib-1.3.1/lib/libz.so (found version "1.3.1")
-- Found PNG: /nix/store/x0qcdj8v7l051ynrzx03d3wlf4pns8s0-libpng-apng-1.6.46/lib/libpng.so (found version "1.6.46")
CMake Warning (dev) at /nix/store/c42964k31qd0xnj5dg72bd4p005w9i1m-cmake-3.31.6/share/cmake-3.31/Modules/FetchContent.cmake:1373 (message):
  The DOWNLOAD_EXTRACT_TIMESTAMP option was not given and policy CMP0135 is
  not set.  The policy's OLD behavior will be used.  When using a URL
  download, the timestamps of extracted files should preferably be that of
  the time of extraction, otherwise code that depends on the extracted
  contents might not be rebuilt if the URL changes.  The OLD behavior
  preserves the timestamps from the archive instead, but this is usually not
  what you want.  Update your project to the NEW behavior or specify the
  DOWNLOAD_EXTRACT_TIMESTAMP option with a value of true to avoid this
  robustness issue.
Call Stack (most recent call first):
  CMakeLists.txt:149 (FetchContent_Declare)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Could NOT find Git (missing: GIT_EXECUTABLE) 
-- Could NOT find Git (missing: GIT_EXECUTABLE) 
-- Configuring done (1.3s)
-- Generating done (0.1s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_C_COMPILER
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025


-- Build files have been written to: /build/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
build flags: -j2 SHELL=/nix/store/k6260bsvzn7n0nsfyjqwn4jndgv063wf-bash-5.2p37/bin/bash
[  0%] Building CXX object src/Utility/CMakeFiles/Utility.dir/src/String.cpp.o
[  0%] Building CXX object src/Core/CMakeFiles/Core.dir/src/BinaryStream.cpp.o
[  0%] Linking CXX static library libUtility.a
[  0%] Built target Utility
[  0%] Building CXX object src/Math/CMakeFiles/Math.dir/src/Trigonometry.cpp.o
[  0%] Building CXX object src/Core/CMakeFiles/Core.dir/src/FileStream.cpp.o
[  0%] Building CXX object src/Math/CMakeFiles/Math.dir/src/Vector.cpp.o
[  0%] Linking CXX static library libMath.a
[  0%] Built target Math
[  0%] Building CXX object src/Core/CMakeFiles/Core.dir/src/MemoryStream.cpp.o
[  1%] Building CXX object src/Core/CMakeFiles/Core.dir/src/Numerics.cpp.o
[  1%] Building CXX object src/Core/CMakeFiles/Core.dir/src/Prng.cpp.o
[  1%] Linking CXX static library libCore.a
[  2%] Generating openloco_text
214+0 records in
214+0 records out
876544 bytes (877 kB, 856 KiB) copied, 0.000987611 s, 888 MB/s
[  2%] Generating openloco_data
78+0 records in
78+0 records out
319488 bytes (319 kB, 312 KiB) copied, 0.000274838 s, 1.2 GB/s
[  2%] Built target Core
3133+0 records in
3133+0 records out
12832768 bytes (13 MB, 12 MiB) copied, 0.00955591 s, 1.3 GB/s
[  2%] Built target segfiles
[  2%] Building CXX object src/Platform/CMakeFiles/Platform.dir/src/Crash.cpp.o
[  2%] Generating openloco_text_section.o
[  3%] Generating openloco_data_section.o
[  3%] Built target linkable_sections
[  3%] Building CXX object src/Platform/CMakeFiles/Platform.dir/src/Platform.Posix.cpp.o
[  4%] Building CXX object src/Platform/CMakeFiles/Platform.dir/src/Platform.Windows.cpp.o
[  4%] Linking CXX static library libPlatform.a
[  4%] Built target Platform
[  5%] Building CXX object src/Diagnostics/CMakeFiles/Diagnostics.dir/src/LogSink.cpp.o
[  5%] Building CXX object src/Diagnostics/CMakeFiles/Diagnostics.dir/src/LogFile.cpp.o
[  5%] Building CXX object src/Diagnostics/CMakeFiles/Diagnostics.dir/src/LogTerminal.cpp.o
[  5%] Building CXX object src/Diagnostics/CMakeFiles/Diagnostics.dir/src/Logging.cpp.o
[  5%] Linking CXX static library libDiagnostics.a
[  5%] Built target Diagnostics
[  5%] Building CXX object src/Interop/CMakeFiles/Interop.dir/src/Hook.cpp.o
[  5%] Building CXX object src/Gfx/CMakeFiles/Gfx.dir/src/PngImage.cpp.o
[  5%] Building CXX object src/Interop/CMakeFiles/Interop.dir/src/Interop.cpp.o
[  6%] Linking CXX static library libGfx.a
[  6%] Built target Gfx
[  7%] Linking CXX static library libInterop.a
[  7%] Built target Interop
[  8%] Building CXX object src/Engine/CMakeFiles/Engine.dir/src/Engine.cpp.o
[  8%] Building CXX object src/Engine/CMakeFiles/Engine.dir/src/Input/ShortcutManager.cpp.o
[  8%] Linking CXX static library libEngine.a
[  8%] Built target Engine
[  9%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Audio/Audio.cpp.o
[  9%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Audio/Channel.cpp.o
[  9%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Audio/OpenAL.cpp.o
[  9%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Audio/VehicleChannel.cpp.o
[ 10%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/CommandLine.cpp.o
[ 10%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Config.cpp.o
[ 10%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Date.cpp.o
[ 10%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Economy/Economy.cpp.o
[ 11%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/EditorController.cpp.o
[ 11%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Effects/Effect.cpp.o
[ 11%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Effects/EffectsManager.cpp.o
[ 11%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Effects/ExhaustEffect.cpp.o
[ 12%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Effects/ExplosionEffect.cpp.o
[ 12%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Effects/ExplosionSmokeEffect.cpp.o
[ 12%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Effects/FireballEffect.cpp.o
[ 13%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Effects/MoneyEffect.cpp.o
[ 13%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Effects/SmokeEffect.cpp.o
[ 13%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Effects/SplashEffect.cpp.o
[ 13%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Effects/VehicleCrashEffect.cpp.o
[ 14%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Entities/Entity.cpp.o
[ 14%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Entities/EntityManager.cpp.o
[ 14%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Entities/EntityTweener.cpp.o
[ 14%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Environment.cpp.o
[ 15%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Game.cpp.o
[ 15%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Airports/CreateAirport.cpp.o
[ 15%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Airports/RemoveAirport.cpp.o
[ 15%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Buildings/CreateBuilding.cpp.o
[ 16%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Buildings/RemoveBuilding.cpp.o
[ 16%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Cheats/Cheat.cpp.o
[ 16%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Company/BuildCompanyHeadquarters.cpp.o
[ 17%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Company/ChangeCompanyColour.cpp.o
[ 17%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Company/ChangeCompanyFace.cpp.o
[ 17%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Company/ChangeLoan.cpp.o
[ 17%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Company/RemoveCompanyHeadquarters.cpp.o
[ 18%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Company/RenameCompanyName.cpp.o
[ 18%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Company/RenameCompanyOwner.cpp.o
[ 18%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Company/UpdateOwnerStatus.cpp.o
[ 18%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Docks/CreatePort.cpp.o
[ 19%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Docks/RemovePort.cpp.o
[ 19%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/GameCommands.cpp.o
[ 19%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/General/LoadSaveQuit.cpp.o
[ 19%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/General/RenameStation.cpp.o
[ 20%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/General/SetGameSpeed.cpp.o
[ 20%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/General/TogglePause.cpp.o
[ 20%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Industries/CreateIndustry.cpp.o
[ 20%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Industries/RemoveIndustry.cpp.o
[ 21%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Industries/RenameIndustry.cpp.o
[ 21%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Road/CreateRoad.cpp.o
[ 21%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Road/CreateRoadStation.cpp.o
[ 22%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Road/RemoveRoadStation.cpp.o
[ 22%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Terraform/ChangeLandMaterial.cpp.o
[ 22%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Terraform/ClearLand.cpp.o
[ 22%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Terraform/CreateTree.cpp.o
[ 23%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Terraform/CreateWall.cpp.o
[ 23%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Terraform/LowerLand.cpp.o
[ 23%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Terraform/LowerRaiseLandMountain.cpp.o
[ 23%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Terraform/LowerWater.cpp.o
[ 24%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Terraform/RaiseLand.cpp.o
[ 24%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Terraform/RaiseWater.cpp.o
[ 24%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Terraform/RemoveTree.cpp.o
[ 24%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Terraform/RemoveWall.cpp.o
[ 25%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Town/CreateTown.cpp.o
[ 25%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Town/RemoveTown.cpp.o
[ 25%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Town/RenameTown.cpp.o
[ 26%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Track/CreateSignal.cpp.o
[ 26%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Track/CreateTrack.cpp.o
[ 26%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Track/CreateTrackMod.cpp.o
[ 26%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Track/CreateTrainStation.cpp.o
[ 27%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Track/RemoveSignal.cpp.o
[ 27%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Track/RemoveTrack.cpp.o
[ 27%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Track/RemoveTrackMod.cpp.o
[ 27%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Track/RemoveTrainStation.cpp.o
[ 28%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/CloneVehicle.cpp.o
[ 28%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/CreateVehicle.cpp.o
[ 28%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/RenameVehicle.cpp.o
[ 28%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehicleChangeRunningMode.cpp.o
[ 29%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehicleOrderDelete.cpp.o
[ 29%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehicleOrderDown.cpp.o
[ 29%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehicleOrderInsert.cpp.o
[ 29%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehicleOrderReverse.cpp.o
[ 30%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehicleOrderSkip.cpp.o
[ 30%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehicleOrderUp.cpp.o
[ 30%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehiclePassSignal.cpp.o
[ 31%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehiclePickup.cpp.o
[ 31%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehiclePickupAir.cpp.o
[ 31%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehiclePickupWater.cpp.o
[ 31%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehiclePlace.cpp.o
[ 32%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehiclePlaceAir.cpp.o
[ 32%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehiclePlaceWater.cpp.o
[ 32%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehicleRearrange.cpp.o
[ 32%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehicleRefit.cpp.o
[ 33%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehicleReverse.cpp.o
[ 33%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehicleSell.cpp.o
[ 33%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameCommands/Vehicles/VehicleSpeedControl.cpp.o
[ 33%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameSaveCompare.cpp.o
[ 34%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/GameState.cpp.o
[ 34%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Graphics/Colour.cpp.o
[ 34%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Graphics/DrawSprite.cpp.o
[ 35%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Graphics/FPSCounter.cpp.o
[ 35%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Graphics/Gfx.cpp.o
[ 35%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Graphics/InvalidationGrid.cpp.o
[ 35%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Graphics/PaletteMap.cpp.o
[ 36%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Graphics/RenderTarget.cpp.o
[ 36%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Graphics/SoftwareDrawingContext.cpp.o
[ 36%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Graphics/SoftwareDrawingEngine.cpp.o
[ 36%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Graphics/TextRenderer.cpp.o
[ 37%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Gui.cpp.o
[ 37%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Input.cpp.o
[ 37%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Input/Keyboard.cpp.o
[ 37%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Input/MouseInput.cpp.o
[ 38%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Input/Shortcuts.cpp.o
[ 38%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Interop/Hooks.cpp.o
[ 38%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Intro.cpp.o
[ 38%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Jukebox.cpp.o
[ 39%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Localisation/Conversion.cpp.o
[ 39%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Localisation/Formatting.cpp.o
[ 39%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Localisation/LanguageFiles.cpp.o
[ 40%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Localisation/Languages.cpp.o
[ 40%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Localisation/StringBuffer.cpp.o
[ 40%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Localisation/StringManager.cpp.o
[ 40%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Localisation/Unicode.cpp.o
[ 41%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Logging.cpp.o
[ 41%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Main.cpp.o
[ 41%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/AnimationManager.cpp.o
[ 41%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/BuildingElement.cpp.o
[ 42%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/IndustryElement.cpp.o
[ 42%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/MapGenerator/HeightMap.cpp.o
[ 42%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/MapGenerator/MapGenerator.cpp.o
[ 42%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/MapGenerator/OriginalTerrainGenerator.cpp.o
[ 43%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/MapGenerator/PngTerrainGenerator.cpp.o
[ 43%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/MapGenerator/SimplexTerrainGenerator.cpp.o
[ 43%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/MapSelection.cpp.o
[ 44%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/RoadElement.cpp.o
[ 44%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/SignalElement.cpp.o
[ 44%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/StationElement.cpp.o
[ 44%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/SurfaceData.cpp.o
[ 45%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/SurfaceElement.cpp.o
[ 45%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/Tile.cpp.o
[ 45%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/TileClearance.cpp.o
[ 45%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/TileLoop.cpp.o
[ 46%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/TileManager.cpp.o
[ 46%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/Track/SubpositionData.cpp.o
[ 46%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/Track/Track.cpp.o
[ 46%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/Track/TrackData.cpp.o
[ 47%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/Tree.cpp.o
[ 47%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Map/WaveManager.cpp.o
[ 47%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Message.cpp.o
[ 47%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/MessageManager.cpp.o
[ 48%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/MultiPlayer.cpp.o
[ 48%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Network/Network.cpp.o
[ 48%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Network/NetworkBase.cpp.o
[ 49%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Network/NetworkClient.cpp.o
[ 49%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Network/NetworkConnection.cpp.o
[ 49%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Network/NetworkServer.cpp.o
[ 49%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Network/Socket.cpp.o
[ 50%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/AirportObject.cpp.o
[ 50%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/BridgeObject.cpp.o
[ 50%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/BuildingObject.cpp.o
[ 50%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/CargoObject.cpp.o
[ 51%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/CliffEdgeObject.cpp.o
[ 51%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/ClimateObject.cpp.o
[ 51%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/CompetitorObject.cpp.o
[ 51%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/CurrencyObject.cpp.o
[ 52%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/DockObject.cpp.o
[ 52%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/HillShapesObject.cpp.o
[ 52%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/IndustryObject.cpp.o
[ 53%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/InterfaceSkinObject.cpp.o
[ 53%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/LandObject.cpp.o
[ 53%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/LevelCrossingObject.cpp.o
[ 53%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/Object.cpp.o
[ 54%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/ObjectImageTable.cpp.o
[ 54%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/ObjectIndex.cpp.o
[ 54%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/ObjectManager.cpp.o
[ 54%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/ObjectStringTable.cpp.o
[ 55%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/RegionObject.cpp.o
[ 55%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/RoadExtraObject.cpp.o
[ 55%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/RoadObject.cpp.o
[ 55%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/RoadStationObject.cpp.o
[ 56%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/ScaffoldingObject.cpp.o
[ 56%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/ScenarioTextObject.cpp.o
[ 56%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/SnowObject.cpp.o
[ 56%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/SoundObject.cpp.o
[ 57%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/SteamObject.cpp.o
[ 57%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/StreetLightObject.cpp.o
[ 57%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/TownNamesObject.cpp.o
[ 58%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/TrackExtraObject.cpp.o
[ 58%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/TrackObject.cpp.o
[ 58%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/TrainSignalObject.cpp.o
[ 58%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/TrainStationObject.cpp.o
[ 59%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/TreeObject.cpp.o
[ 59%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/TunnelObject.cpp.o
[ 59%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/VehicleObject.cpp.o
[ 59%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/WallObject.cpp.o
[ 60%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Objects/WaterObject.cpp.o
[ 60%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/OpenLoco.cpp.o
[ 60%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/Paint.cpp.o
[ 60%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintAirport.cpp.o
[ 61%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintBridge.cpp.o
[ 61%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintBuilding.cpp.o
[ 61%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintDocks.cpp.o
[ 61%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintEffectEntity.cpp.o
[ 62%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintEntity.cpp.o
[ 62%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintIndustry.cpp.o
[ 62%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintRoad.cpp.o
[ 63%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintRoadStation.cpp.o
[ 63%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintSignal.cpp.o
[ 63%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintStation.cpp.o
[ 63%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintSurface.cpp.o
[ 64%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintTile.cpp.o
[ 64%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintTileDecorations.cpp.o
[ 64%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintTrack.cpp.o
[ 64%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintTrainStation.cpp.o
[ 65%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintTree.cpp.o
[ 65%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintVehicle.cpp.o
[ 65%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Paint/PaintWall.cpp.o
[ 65%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Random.cpp.o
[ 66%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/S5/S5.cpp.o
[ 66%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/S5/SawyerStream.cpp.o
[ 66%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Scenario.cpp.o
[ 67%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/ScenarioConstruction.cpp.o
[ 67%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/ScenarioManager.cpp.o
[ 67%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/ScenarioObjective.cpp.o
[ 67%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/ScenarioOptions.cpp.o
[ 68%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/SceneManager.cpp.o
[ 68%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Title.cpp.o
[ 68%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Tutorial.cpp.o
[ 68%] Building CXX object src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Ui.cpp.o
In file included from /build/source/src/OpenLoco/src/Ui.cpp:29:
/nix/store/mdv8j3xan76z873nqqvj01y4psray2y2-sdl2-compat-2.32.52-dev/include/SDL2/SDL_syswm.h:68:10: fatal error: X11/Xlib.h: No such file or directory
   68 | #include <X11/Xlib.h>
      |          ^~~~~~~~~~~~
compilation terminated.
make[2]: *** [src/OpenLoco/CMakeFiles/OpenLoco.dir/build.make:3243: src/OpenLoco/CMakeFiles/OpenLoco.dir/src/Ui.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:637: src/OpenLoco/CMakeFiles/OpenLoco.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.14.2, NixOS, 25.05 (Warbler), 25.05.20250412.2631b0b`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.1`
 - nixpkgs: `/nix/store/5ds20jm3x2s4z7wn3581r6lc9ybmh45b-source`

### Notify maintainers

@icewind1991

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/games/openloco/default.nix
+++ b/pkgs/games/openloco/default.nix
@@ -11,6 +11,7 @@
   pkg-config,
   yaml-cpp,
   fmt,
+  xorg,
 }:
 let
   sfl-src = fetchFromGitHub {
@@ -68,6 +69,7 @@ stdenv.mkDerivation rec {
     openal
     yaml-cpp
     fmt
+    xorg.libX11
   ];
 
   meta = {

